README.md

```diff
@@ -0,0 +1,17 @@
+# Branches in NorESMhub/CTSM
+
+This repository has the following CTSM branches working with NorESM2:
+
+- *master* - keep track with the master branch of [ESCOMP/CTSM](https://github.com/ESCOMP/CTSM).
+  - This branch is used by [NorESMhub/NorESM_LandSites_Platform](https://github.com/NorESMhub/NorESM_LandSites_Platform)
+  - This branch can be used for running the most recent version of [FATES](https://github.com/NGEET/fates)
+
+- *release-clm5.0.14-Nor* - The working branch used by NorESM2. Currently, all the pull requests should be sent to this branch. 
```

Comment:
I would suggest specifying that only PRs that are not answer changing should be sent to this branch. New developments should be sent elsewhere. Currently there is a clm-nor-dev, which should be used for ongoing developments, but I would also suggest renaming this.